lom/src/plugins/sonic/plugin/detection/link/linkcrc/link_crc.go

```diff
@@ -2,6 +2,7 @@
 package linkcrc
```

Comment:
Line 224:  if localTimeStampUtc.Sub(outlier.TimeStamp).Seconds() < float64(lookBackPeriodInSecs) {
You may add "else" clause to break as rest of the points are ahead of the interval

Reply:
Done.